test/alice/handlers/help_test.exs

```diff
@@ -4,7 +4,16 @@ defmodule Alice.Handlers.HelpTest do
   alias Alice.Handlers.TestHandler
 
   test "help_for_handler generates the correct output" do
-    assert Help.help_for_handler(TestHandler) ==
+    conn = Alice.Conn.make(%{}, %{me: %{name: "alice"}})
+    assert Help.help_for_handler(TestHandler, conn) == test_help_message_output(conn)
+  end
+
+  test "help_for_handler generates the correct output with a different bot name" do
+    conn = Alice.Conn.make(%{}, %{me: %{name: "mad_hatter"}})
+    assert Help.help_for_handler(TestHandler, conn) == test_help_message_output(conn)
+  end
+
+  defp test_help_message_output(%Alice.Conn{slack: %{me: %{name: bot_name}}}) do
```

Comment:
Hmm maybe this should just use the `Conn.bot_name` on the passed in `Conn`, instead of deconstructing using the `slack` key again?